controllers/ListItemController.php

```diff
@@ -17,6 +17,34 @@
 
 	class ListItemController
 	{
+		/**
+			* Swagger doc block
+			*
+			* @OA\Get(
+			* path="ListItem/get_list_items",
+			* summary="Find list items by list id",
+			* @OA\Parameter(
+			*		name="listID",
+			* 	in= "query",
+			* 	description="Identifier of the list to fetch items from",
+			* 	required=true,
+			* 	type="string",
+			* 	style="form"
+			*	),
+			* @OA\Response(
+			*		response=200,
+      *   description="successful operation",
+			*		@OA\Schema(
+ 			*   	type="array",
+ 			*  		@OA\Items(ref="#/components/schemas/Item")

```

Comment:
I don't see the `Item` schema defined anywhere for this response

Reply:
I am not sure how to document this. Can you help please

Reply:
A good example of how to create a scheme for an entity would can be found [here](https://github.com/gigili/good-food-tracker-api/blob/aafb302496a171091bb2fcf7b19e36225496459d/src/Entity/UserEntity.php#L16-L26)

```php
/**
* @OA\Schema (
*    schema="UserEntity",
*    properties={
*     @OA\Property(property="id", type="string"),
*    }
* )
*/
 ```

For your case the `schama="UserEntity"` would change to `schema="ListItem"` and than later on when you want to reference that schema in other parts of documentation you can use  `ref="#/components/schemas/ListItem"`

Reply:
I have made this change